frontend/packages/helm-plugin/src/catalog/utils/const.ts

```diff
@@ -0,0 +1,4 @@
+// Annotation for human readable chart name, ready to be displayed in UI
+export const CHART_NAME_ANNOTATION = 'helm-chart.openshift.io/name';
+// Annotation for chart category, e.g. redhat, partner, community
```

Comment:
Approved too soon, where are we getting "redhat" - can we make this "Red Hat" ?

Reply:
I think the well known values should be translated. Any other unknown value can be used as is.
This localize `partner` and `community` and also address the `redhat` => `Red Hat`

Reply:
Have updated the PR to use localised string for well known values and capitalize the rest.